add_kdrama.php

```diff
@@ -0,0 +1,26 @@
+<?php
+require_once "functions.php";
+
+var_dump($_POST);
```

Comment:
I think you can get rid of this as it's just for your own checks

Reply:
Make sure you do remove this var_dump line entirely, we only use this four our own debugging.